Flutter/datagrid/filtering.md

```diff
@@ -1325,4 +1325,86 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
\ No newline at end of file
+<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
+
+## Set a StrongDataType for filter behavior

```

Comment:
## Perform filtering to the user-defined type

Reply:
Modified the title